docs/reference/api/docker_remote_api_v1.15.md

```diff
@@ -196,7 +199,7 @@ Json Parameters:
 -   **Tty** - Boolean value, Attach standard streams to a tty, including stdin if it is not closed.
 -   **OpenStdin** - Boolean value, opens stdin,
 -   **StdinOnce** - Boolean value, close stdin after the 1 attached client disconnects.
--   **Env** - A list of environment variables in the form of `VAR=value`
+-   **Env** - A list of environment variables in the form of `["VAR=value"[,"VAR2=value2"]]`
```

Comment:
Maybe good to state.  "The outer brackets are required for JSON. The inner brackets are syntactical to indicate multiple values are an option."